core/src/main/java/org/stellar/anchor/sep24/Sep24Service.java

```diff
@@ -555,4 +572,74 @@ public InfoResponse getInfo() {
                 sep24Config.getFeatures().getClaimableBalances()))
         .build();
   }
+
+  private void validatedAndPopulateQuote(
```

Comment:
I wrote a utility class that does this validation for SEP-6. Do you think we can leverage it here? I think the logic should be the same. https://github.com/stellar/java-stellar-anchor-sdk/blob/8e595eab7aa01147d610cb39313afa0c18d75630/core/src/main/java/org/stellar/anchor/sep6/ExchangeAmountsCalculator.java#L31-L62